kubemarine/procedures/migrate_cri.py

```diff
@@ -216,7 +209,12 @@ def _migrate_cri(cluster: KubernetesCluster, node_group: dict):
                                 "sudo systemctl restart containerd && "
                                 # start kubelet
                                 "sudo systemctl restart kubelet")
-        control_plane["connection"].sudo(f"sudo kubectl uncordon {node['name']}", is_async=False, hide=False)
+
+        if "control-plane" in node["roles"]:
+            kubernetes.wait_uncordon(node["connection"])
+        else:
+            control_plane["connection"].sudo(f"kubectl uncordon {node['name']}", is_async=False, hide=False)
```

Comment:
I tried `migrate_cri` on dev cluster with 1 balancer, 2 control planes, and 1 worker. Though it is not supported scheme, it is one of the minimal schemes to test upgrade with separate control planes and workers.

It was stably failed with different connection errors. The same problem was with the new CRI upgrade procedure in `migrate_kubemarine`. So I fixed both of them the same way using `kubernetes.wait_uncordon` (see its implementation).

Probably not being actual for real clusters, it still simplifies dev-testing of both procedures.